client/src/main/java/com/paypal/selion/reports/runtime/Gatherer.java

```diff
@@ -41,7 +44,7 @@ static String saveGetLocation(WebDriver driver) {
                 location = driver.getCurrentUrl();
             }
         } catch (Exception exception) {
-            logger.log(Level.WARNING, "Current location couldn't be retrieved by getCurrentUrl().", exception);
+            logger.log(Level.FINEST, "Current location couldn't be retrieved by getCurrentUrl().", exception);
```

Comment:
currentLocation is valid for browser testing on mobileapps such as when the user is using safari app on appium-ios.  If a user is testing a nativeapp such as InternationalMountains.app, an exception will always be thrown on the call getCurrentURL.  I didnt want red text printed to the console for this scenario when testing native apps so i changed it to finest.  I didnt choose fine because i didnt want to clutter up the logs who is by default logging fine.